jackal_description/urdf/configs/front_flea3

```diff
@@ -0,0 +1,2 @@
+
+JACKAL_FLEA3=1
```

Comment:
Consider adding an example for laser + camera. That was an intended use-case to be covered by this bracket design.

It won't be possible using the envvar scheme, but it would be fun/useful to also include a URDF which demonstrates how to set up a more sophisticated robot using these macros— eg, front and back BB2 units tilted down + LIDAR + upper bridge/deck with a forward-facing flea3 on it.